jaggery-modules/rxt/module/scripts/app/app.js

```diff
@@ -980,6 +980,19 @@ var app = {};
         }
         return details.enabled ? details.enabled : false;
     };
+
+    app.isTaxonomyEnabled = function () {
+        var TaxonomyService = carbon.server.osgiService('org.wso2.carbon.governance.taxonomy.services.TaxonomyService');
+        var availability = TaxonomyService.getTaxonomyAvailability();
+        availability = String(availability);
+        availability = (availability == "true");
```

Comment:
We can get rid of line 988 to 993 by doing this:
return (availabity == 'true');